packages/twenty-server/src/database/typeorm/core/migrations/billing/1734532875877-dropSubscriptionCustomerConstraint.ts

```diff
@@ -0,0 +1,19 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class DropSubscriptionCustomerConstraint1734532875877
+  implements MigrationInterface
+{
+  name = 'DropSubscriptionCustomerConstraint1734532875877';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      `ALTER TABLE "core"."billingSubscription" DROP CONSTRAINT "FK_9120b7586c3471463480b58d20a"`,
```

Comment:
this should be removed from the original migration, otherwise we will apply it first before removing it and end up in the same issue while applying it